packages/peregrine/lib/talons/MagentoRoute/getRouteComponent.js

```diff
@@ -25,7 +25,7 @@ const getRouteComponent = async (apiBase, pathname) => {
             throw new Error('404');
         }
 
-        const { type, id } = resolvedRoute;
+        const { type, id, redirectCode, relative_url } = resolvedRoute;
```

Comment:
Very minor, but I have a preference to alias on destructure to be consistent with variable naming. I don't think I would write `const relative_url = resolvedRoute.relative_url` if we were missing the sugar.

```suggestion
        const { type, id, redirectCode, relative_url: relativeUrl } = resolvedRoute;
```

Reply:
I didn't end up changing this one not because I disagree necessarily, but because the underscores in variable names isn't how we generally name variables in our codebase. Basically, we're getting a name from GQL and turning it into something our codebase is going to use, and this feels like the best place to make that transition.